### ISSUE TITLE
Display the installed status of a package without having to click on it

### ISSUE THREAD

**Issue by pappfer** [solus-project/solus-sc#45]
At the moment when I'm looking for a package in Software Center I can't tell whether I installed it already or not until I click on the package itself. 

It would be nice to have a simple indicator somewhere next to the package name so that we could tell whether that package was installed without having to click on it.

![software-center-1](https://cloud.githubusercontent.com/assets/738623/20496713/990ac5aa-b026-11e6-8b7e-76344b58cf91.png)
